for_or_against.py

```diff
@@ -240,21 +242,21 @@ def save_analysis(analysis: AnalysisOutput, output_path: str = "stance_analysis_
         json.dump(analysis.model_dump(), f, indent=2)
 
 
-if __name__ == "__main__":
-    import argparse
-
-    parser = argparse.ArgumentParser(description='Analyze transcript stances')
-    parser.add_argument('--transcript', default='transcript.json', help='Path to transcript JSON file')
-    parser.add_argument('--limit', type=int, help='Number of speakers to analyze')
-    parser.add_argument('--output', default='stance_analysis_results.json', help='Output JSON file path')
-
-    args = parser.parse_args()
-
+@click.command()
```

Comment:
Making some small incremental changes.
For instance - using click instead of arg parse to better match the rest of the code in the repo